File: scommons-client/src/main/react/components/AppBrowsePanel/AppBrowsePanel.js
const React = require('react')
const styles = require('./AppBrowsePanel.css')
const BrowseTreePanel = require('../BrowseTreePanel/BrowseTreePanel')

class AppBrowsePanel extends React.Component {
  render() {
    return (
      <div className="row-fluid">
        <div className="span4">
          <div className="well sidebar-nav">
            {//<ButtonsPanel ui:field="buttonsPanel" className={style.sidebar-bp} />
            }
            <BrowseTreePanel/>
          </div>
        </div>
        <div className="span8" />
      </div>
    )
  }
}

module.exports = AppBrowsePanel

File: ui/src/main/scala/scommons/client/app/AppBrowsePanel.scala
package scommons.client.app

import scommons.client.ui.tree.{BrowseTree, BrowseTreeProps}
import scommons.client.ui.{ButtonsPanel, ButtonsPanelProps}
import scommons.react._

case class AppBrowsePanelProps(buttonsProps: ButtonsPanelProps,
                               treeProps: BrowseTreeProps)

object AppBrowsePanel extends FunctionComponent[AppBrowsePanelProps] {

  protected def render(compProps: Props): ReactElement = {
    val props = compProps.wrapped

    <.div(^.className := "row-fluid")(
      <.div(^.className := "span4")(
        <.div(^.className := "well sidebar-nav")(
          <.div(^.className := AppBrowsePanelCss.sidebarBp)(
            <(ButtonsPanel())(^.wrapped := props.buttonsProps)()
          ),
          <(BrowseTree())(^.wrapped := props.treeProps)()
        )
      ),
      <.div(^.className := "span8")(
        compProps.children
      )
    )
  }
}
